tests/test_bgp_neighbor.rb

```diff
@@ -316,26 +423,41 @@ def test_bgpneighbor_set_get_remote_as
     end
   end
 
-  def test_bgpneighbor_set_get_remove_private_as_options
+  def test_set_get_remove_private_as_options
     %w(default test_vrf).each do |vrf|
-      neighbor = RouterBgpNeighbor.new(@@asn, vrf, @@addr)
-      options = [:enable, :disable, :all, :"replace-as", 'enable', 'disable',
-                 'all', 'replace-as', neighbor.default_remove_private_as]
+      neighbor = create_neighbor(vrf)
+      options = []
+      if platform == :ios_xr
+        assert_raises(Cisco::UnsupportedError) do
+          options = [:enable, :disable, :all, :"replace-as", 'enable',
+                     'disable', 'all', 'replace-as',
+                     neighbor.default_remove_private_as]
```

Comment:
This is really just: ```assert_raises(Cisco::UnsupportedError) { neighbor.default_remove_private_as }``` right?

Note that this assertion will fail once we rebase to pick up the change from #82 - this will need to change to ```assert_nil``` at that point.